src/theme.c

```diff
@@ -548,6 +551,12 @@ entry(struct theme *theme, const char *key, const char *value)
 	if (match_glob(key, "osd.window-switcher.item.active.border.width")) {
 		theme->osd_window_switcher_item_active_border_width = atoi(value);
 	}
+	if (match_glob(key, "osd.workspace-switcher.boxes.width")) {
+		theme->osd_workspace_switcher_boxes_width = atoi(value);
+	}
+	if (match_glob(key, "osd.workspace-switcher.boxes.height")) {
+		theme->osd_workspace_switcher_boxes_height = atoi(value);
+	}
```

Comment:
Maybe we should additionally check If one of these is `0` and then set the other one to `0` as well.
This could happen in `post_processing()`

Otherwise I think only setting `osd.workspace-switcher.boxes.width` to `0` but keeping `.height` as something different will mess with the height calculations (we disable the boxes but still use their height for calculating the workspace name offset).

Other than that this LGTM.

Reply:
Thanks, addressed it in this to-be-squashed commit: https://github.com/labwc/labwc/pull/1299/commits/c6f6ebe96694d9765762062158971b78f11686cf